index.html

```diff
@@ -1006,6 +1008,109 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `protocol_handlers` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-dfn-for=
+          "manifest">protocol_handlers</dfn></code> member is an <a>array</a>
+          of <a>protocol handler description</a>s that allows a web application
+          to handle URL protocols.
+        </p>
+        <p class="example">
+          Protocol handlers could, for instance, be used for web app
+          communication where one app directly invokes another and passes data
+          via custom protocol links.
+        </p>
+        <p>
+          How protocol handlers are presented, and how many of them are shown
+          to the user, is at the discretion of the user agent and/or operating
+          system.
+        </p>
+        <p>
+          To <dfn>process the `protocol_handlers` member</dfn>, given
+          [=object=] |json:JSON|, |manifest:ordered map|, and |manifest
+          URL:URL|:
+        </p>
+        <ol>
+          <li>Let <var>processedProtocolHandlers</var> be a new [=list=].
+          </li>
+          <li>[=list/For each=] <var>protocol_handler</var> (<a>protocol
+          handler description</a>):
+            <ol>
+              <li>If |protocol_handler|["protocol"] or
+              <var>protocol_handler</var>["url"] is undefined,
+              [=iteration/continue=].
+              </li>
+              <li>Let (<dfn>normalizedProtocol</dfn>, <dfn>normalizedUrl</dfn>)
+              be the result of running [=normalize protocol handler
+              parameters=] with <var>protocol_handler</var>["protocol"], <var>
+                protocol_handler</var>["url"] using <var>manifest URL</var> as
+                the base URL, and [=this=]'s relevant [=environment settings
+                object=]. If the result is failure, [=iteration/continue=].
+              </li>
+            </ol>
+          </li>
+          <li>If [=normalizedUrl=] is not [=manifest/within scope=] of
+          <var>manifest URL</var>, [=iteration/continue=].
+          </li>
+          <li>If [=normalizedUrl=] already exists in
+          <var>processedProtocolHandlers</var>, [=iteration/continue=].
+          </li>
+          <li>[=List/Append=] |protocol_handler| to
+          <var>processedProtocolHandlers</var>.
+          </li>
+        </ol>
+        <ul>
```

Comment:
Formatting: Why is there a `<ul>` here with one `li` in it, and then a bunch of other things, then a `</ul>`? Was it meant to be closed after the `li` but you forgot, and then tidy-html added it way down there?

Reply:
seems like it indeed. I fixed the list since I hadn't noticed this one! Thanks